### PR TITLE
format with .uncrustify.cfg

### PR DESCRIPTION
**Pull request by wangxufire**
format all C source codes with uncrustify.

1. install uncrustify on your system.
2. run code-format.sh (on unix like) or code-format.bat (on windows) in project root directory.

### CODE REVIEW

**Comment by madeye** [comment]
Don't perform uncrustify on any third-party source code, including json.c etc.

**Comment by wangxufire** [comment]
I just included the src directory, not include the submodules.

**Comment by madeye** [comment]
Please exclude base64.*, json.* and uthash.c in src folder. They are code forked from other projects.

**Comment by madeye** [comment]
Merged, thanks!

File: src/local.c [hunk 1]
```diff
@@ -1664,16 +1666,16 @@ main(int argc, char **argv)
     // Setup UDP
     if (mode != TCP_ONLY) {
         LOGI("udprelay enabled");
-        char *host = remote_addr[0].host;
-        char *port = remote_addr[0].port == NULL ? remote_port : remote_addr[0].port;
+        char *host                       = remote_addr[0].host;
```
Comment (pexcn) [inline src/local.c]: 这里为什么要加那么多空格？是什么风格呢？

Comment (wangxufire) [reply src/local.c]: 可能配的规则有类似与最后的等号对其的吧，可以把这个三目运算改用if else。

Comment (wangxufire) [reply src/local.c]: 手机上看错了，这个是和`struct sockaddr_storage *storage =` 对齐。

Comment (pexcn) [reply src/local.c]: 不是很懂，感觉还是 `=` 左右有 1 个空格就好了

**Pull request merged**